src/background.js

```diff
@@ -32,7 +33,7 @@ chrome.browserAction.onClicked.addListener(function () {
     chrome.tabs.query({active: true, currentWindow: true}, function (tabs) {
         if (tabs.length > 0) {
             let url = tabs[0].url;
-            if (url === workManagerURL || url === workManagerURL2) {
+            if (url.indexOf(workManagerURL) !== -1 || url.indexOf(workManagerURL2) !== -1 || url.indexOf(workManagerURL3) !== -1) {
```

Comment:
```js
url.indexOf(workManagerURL) !== -1
url.indexOf(workManagerURL) > -1
url.includes(workManagerURL)
```
The 3 examples all do the same thing. Nothing wrong with your version, just showing you other options.